components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/ClaimMetadataManagementServiceImpl.java

```diff
@@ -300,16 +309,16 @@ public void removeExternalClaim(String externalClaimDialectURI, String externalC
             throws ClaimMetadataException {
 
         if (StringUtils.isBlank(externalClaimDialectURI)) {
-            throw new ClaimMetadataException("External claim dialect URI cannot be empty");
+            throw new ClaimMetadataClientException(ERROR_CODE_EMPTY_EXTERNAL_DIALECT_URI.getCode(),
+                    "External claim dialect URI cannot be empty");
```

Comment:
This error message is almost similar to the "ERROR_CODE_EMPTY_EXTERNAL_DIALECT_URI" error message hence did not add a new error message in the enum.